src/flows/CostCalculator/CostCalculator.tsx

```diff
@@ -1,48 +1,27 @@
 import React from 'react';
 import { useForm } from 'react-hook-form';
 
-import type {
-  CostCalculatorEstimateParams,
-  CostCalculatorEstimateResponse,
-  EmploymentTermType,
-} from '@/src/client';
+import type { CostCalculatorEstimateResponse } from '@/src/client';
 import { Form } from '@/src/components/ui/form';
 
 import { JSONSchemaFormFields } from '@/src/components/form/JSONSchemaForm';
 import { useValidationFormResolver } from '@/src/components/form/yupValidationResolver';
 import { Button } from '@/src/components/ui/button';
 import { Disclaimer } from '@/src/flows/CostCalculator/Disclaimer';
-import { useCostCalculator } from '@/src/flows/CostCalculator/hooks';
-import { convertToCents } from '@/src/lib/utils';
-
-type FormValues = {
-  currency: string;
-  country: string;
-  salary: string;
-} & Partial<{
-  region: string;
-  age: number;
-  contract_duration_type: EmploymentTermType;
-}>;
+import {
+  defaultEstimationOptions,
+  useCostCalculator,
+} from '@/src/flows/CostCalculator/hooks';
+import {
+  CostCalculatorEstimateFormValues,
+  CostCalculatorEstimationOptions,
+} from './types';
 
 type CostCalculatorProps = Partial<{
   /**
    * Estimation params allows you to customize the parameters sent to the /cost-calculator/estimation endpoint.
    */
-  estimationParams: Partial<{
-    /**
-     * Title of the estimation. Default is 'Estimation'.
-     */
-    title: string;
-    /**
-     * Include benefits in the estimation. Default is false.
-     */
-    includeBenefits: boolean;
-    /**
-     * Include cost breakdowns in the estimation. Default is false.
-     */
-    includeCostBreakdowns: boolean;
-  }>;
+  estimationOptions?: CostCalculatorEstimationOptions;
```

Comment:
IMO `estimationOptions` is slightly better name than `estimationParams`. lmk your thoughts